CONTRIBUTING.rst

```diff
@@ -25,112 +25,97 @@ Thank you for your interest in contributing to ivadomed! This project uses the f
 
 .. _installation_contributor:
 
-Contributor or Developer ``ivadomed`` installation 
+Developer ``ivadomed`` installation
 ++++++++++++++++++++++++++++++++++++++++++++++++++
 
+1. Download the code:
+
+    ::
+
+        cd ivadomed
+        git clone https://github.com/ivadomed/ivadomed.git
+
+2. Isolate your work in a virtual environment:
+
+    We recommend developing inside of ``venv`` or ``conda``.
+    It makes debugging easier.
+
     .. tabs::
-        
-        .. tab:: NVIDIA GPU Support
-
-            PyTorch, an integral part of ``ivadomed``, ships 
-            CUDA 10.2 and CUDA 11.1 runtime by default with its
-            respective installation binaries.
-
-            In case if you're wondering about CUDA runtime, Ampere-based GPUs
-            (with a `Compute Capability <https://developer.nvidia.com/cuda-gpus>`_
-            of 8.x) only work with CUDA>=11.1. Although CUDA 11.1 is
-            backward compatible with older hardware, CUDA 10.2 is
-            preferred if available.
-
-            Thus, to accelerate ``ivadomed`` with CUDA 10.2 on a Linux system, you'd
-            need to have GPUs installed with an `NVIDIA driver version >=440.33 
-            <https://docs.nvidia.com/deploy/cuda-compatibility/index.html#minor-version-compatibility>`_.
-            And, for CUDA 11.1 you'd rather need an upgraded NVIDIA driver version >=450.
-            
-            To verify the NVIDIA driver version, look in ``/sys`` by executing the
-            command:
 
-            ::
+        .. group-tab:: ``venv``
 
-                cat /sys/module/nvidia/version
-                
-            and it will return your current driver version.
-            
-            Before proceeding with the installation, we suggest you to set up a virtual environment
-            by following the instructions as specified in the :ref:`step 1 of the installation <installation_step1>`.
+            Create the environment:
 
-            Once you've set up a virtual environment and activated it, we recommend installing 
-            ``ivadomed`` from source along with some additional dependencies related to building
-            documentation, linting and testing:
+            .. code::
 
-            .. tabs::
+                python -m venv venv
```

Comment:
Maybe

```suggestion
                python -m venv ivadomed_venv
```

but then that makes

```
ivadomed/ivadomed_venv
```

which is weird. And also `.gitignore` *already had an entry* for `venv/`. So if we change this we should change that too.